warehouse/models/mart/ntd/_mart_ntd.yml

```diff
@@ -0,0 +1,41 @@
+version: 2
+
+models:
+  - name: dim_annual_database_agency_information
+    description: |
+      Versioned extracts of the NTD Annual Database Agency Information.
```

Comment:
Doesn't this only contain 2021 right now? I think it is better to be clear about what is actually in here. If we automate this we can update it but presumably that will require other updates anyway.

Reply:
Ah I was going to import some others too; specifically 2018 at least since that's what is in Airtable. The key on `dim_organizations` only points to 2021. If you want to limit this to 2021, I can do that.

Reply:
We already have `mart_transit_database.dim_ntd_agency_info` containing the 2018 data. And I guess maybe that makes me realize we should just add this in there...? But that is confusing because right now that is structured as an Airtable-sourced table. If you bring 2018 data in this table let's deprecate `mart_transit_database.dim_ntd_agency_info`

Reply:
Right I was planning on deprecating the Airtable table; I don't think we actually want/need it in the broader Airtable framework.